### PR TITLE
Release v2.0.2 PR

### PR DESCRIPTION
**Pull request by matthdsm**
- Drop unsed params
- Set aligner to `star` for RNA-seq
- Finetune resources
- Fix some bugs for different input tags
- Update modules

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -30,8 +30,7 @@ jobs:
     strategy:
       matrix:
         NXF_VER:
-          - "24.04.2"
-          - "latest-everything"
```
Comment (nvnieuwk) [inline .github/workflows/ci.yml:34]: No more tests on the latest version?

**Review by nvnieuwk — APPROVED** [review]
One small comment, LGTM otherwise

File: .github/workflows/ci.yml (hunk 1 above)
Comment (matthdsm) [reply .github/workflows/ci.yml:34]: eh 🤷🏻  it's all overhead for not much profit

Comment (nvnieuwk) [reply .github/workflows/ci.yml:34]: Allright! as long as it works on the version we are using :p

**Pull request merged**